components/src/molecules/rating/Rate.stories.js

```diff
@@ -0,0 +1,76 @@
+import { storiesOf } from '@storybook/vue'
+import { action } from '@storybook/addon-actions'
+
+import AlpacaRate from './Rate.vue'
```

Comment:
Folder name is `rating` and component `Rate`, can we use only one name for both?